docs/HOWTO -- NVIDIA AGX ORIN  on-device kernel build.md

```diff
@@ -0,0 +1,112 @@
+# HOWTO -- NVIDIA AGX ORIN on kernel on-device development process
+
+
+Nvidia is powerful device so developing and something lowlevel as drivers and passthroughs may be fast and efficient when done on board directly.
+
+
+1. Preparational steps -- get a device, plug in monitor and keyboard (in most cases you need DP cable and monitor with DP socket, adapters DP-HDMI will not work before firmware update).
+2. Boot and update onboard L4T ubuntu based linux distribution -- here are the link to the nvidia instructions -- .
+3. Get a M.2 SSD nvme drive ( 512Gb or more is recommended ) and install it in the slot on the bottom of the device.
+You'll have something like 
+```
+[    7.686537] nvme 0004:01:00.0: Adding to iommu group 11
+[    7.691689] nvme nvme0: pci function 0004:01:00.0
+[    7.696285] nvme 0004:01:00.0: enabling device (0000 -> 0002)
+[    7.761026] nvme nvme0: 12/0/0 default/read/poll queues
+[    7.780966]  nvme0n1: p1
+```
+in the logs after reboot
+
+Create a partition table and a ext4 partition.
+Mount the partition, copy the contents of the /home directory to the new drive.
```

Comment:
**, and then copy...**

Please use '' for /home: '/home'
https://www.markdownguide.org/basic-syntax/#code